src/android/DBHelper.java

```diff
@@ -80,6 +88,23 @@ else if (v == 2){//BTE:
                     e.printStackTrace();
                 }
             }
+            else if (v == 3){//GPS:
+
+                JSONObject subObject = new JSONObject();
+                try {
+                    subObject.put("identifier", cursor.getString(1));
```

Comment:
this produces something like this

```
{
  "identifier" : ....,
  "Enter/Exit": ....,
  "lat": ....,
  "lon": ....,
  "alt": ....,
  "timestamp": ....,
}
```

while we need this according to the specs:

```
{ 
    timestamp: 'yyyymmddThhmmss+000' ,
    source: “ios”,
    gps: { lat: , lon:, alt: , timestamp: yyymmddThhmmss+000 } // for location monitoring
    region: { “identifier” : “enter | exit” } // for region monitoring
},
```

note this for one device in the `data` element. here is an example of what a final POST looks like for many devices:

```
{ 
	"choice" : ["scan"],
	  "data" : [
			{ 
				timestamp: 'yyyymmddThhmmss+000' ,
				source: “ios”,
				gps: { lat: , lon:, alt: , timestamp: yyymmddThhmmss+000 } // for location monitoring
				region: { “identifier” : “enter | exit” } // for region monitoring
			},
			{ 
				timestamp: 'yyyymmddThhmmss+000' ,
				source: “ios”,
				gps: { lat: , lon:, alt: , timestamp: yyymmddThhmmss+000 } // for location monitoring
				region: { “identifier” : “enter | exit” } // for region monitoring
			},
		    ....,
	  ],
	"comment" : "test",
	"poll" : "/api/v1/polls/poll/rfscan/"
}
```